From 42fe55d38b7b9e838d1962da6a717f765fb055d0 Mon Sep 17 00:00:00 2001
Message-Id: <42fe55d38b7b9e838d1962da6a717f765fb055d0.1389014116.git.minovotn@redhat.com>
In-Reply-To: <c8cc35838d42aa286242772d97e3a9be7bb786ba.1389014116.git.minovotn@redhat.com>
References: <c8cc35838d42aa286242772d97e3a9be7bb786ba.1389014116.git.minovotn@redhat.com>
From: Paolo Bonzini <pbonzini@redhat.com>
Date: Mon, 9 Dec 2013 14:09:01 +0100
Subject: [PATCH 13/50] block/raw: copy BlockLimits on raw_open

RH-Author: Paolo Bonzini <pbonzini@redhat.com>
Message-id: <1386598178-11845-16-git-send-email-pbonzini@redhat.com>
Patchwork-id: 56054
O-Subject: [RHEL 7.0 qemu-kvm PATCH 15/52] block/raw: copy BlockLimits on raw_open
Bugzilla: 1007815
RH-Acked-by: Jeffrey Cody <jcody@redhat.com>
RH-Acked-by: Fam Zheng <famz@redhat.com>
RH-Acked-by: Stefan Hajnoczi <stefanha@redhat.com>

From: Peter Lieven <pl@kamp.de>

Signed-off-by: Peter Lieven <pl@kamp.de>
Signed-off-by: Stefan Hajnoczi <stefanha@redhat.com>
(cherry picked from commit 04f19e4d2da4325297c99e94f20b0aca52546209)

Conflicts:
	block/raw_bsd.c [replaced by block/raw.c]
---
 block/raw.c | 1 +
 1 file changed, 1 insertion(+)

Signed-off-by: Michal Novotny <minovotn@redhat.com>
---
 block/raw.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/block/raw.c b/block/raw.c
index 67d6445..c853258 100644
--- a/block/raw.c
+++ b/block/raw.c
@@ -7,6 +7,7 @@ static int raw_open(BlockDriverState *bs, QDict *options, int flags,
                     Error **errp)
 {
     bs->sg = bs->file->sg;
+    bs->bl = bs->file->bl;
     return 0;
 }
 
-- 
1.7.11.7

